android/src/main/java/com/google/android/cameraview/CameraView.java

```diff
@@ -311,7 +313,9 @@ public void setUsingCamera2Api(boolean useCamera2) {
             }
             mImpl = new Camera1(mCallbacks, mImpl.mPreview, mBgHandler);
         }
-        start();
+        if(wasOpened){
```

Comment:
The following change is causing issues on devices that fallback to Camera1 even when Camera2 is used.... Implementing further fixes for those.

Reply:
Changes done.